src/IECoreMaya/FromMayaMeshConverter.cpp

```diff
@@ -556,5 +643,24 @@ IECoreScene::PrimitivePtr FromMayaMeshConverter::doPrimitiveConversion( MFnMesh
 		}
 	}
 
+	bool convertCreases = creasesParameter()->getTypedValue();
```

Comment:
you should grab the value from the `operands` rather than the parameter itself. i'm sure other parts of this file are not doing that either

Reply:
After talking to Andrew we decided to leave this one as is for now. We should probably clean it up at some point as it's technically more correct to use the `operands`.